charm/docs/tutorial/test-netbox.md

```diff
@@ -0,0 +1,54 @@
+# Testing NetBox
+
+We recommend the [Getting Started Tutorial](./getting-started.md) to get familiarised with NetBox in a Juju deployment.
+
+However, if your goal is to test NetBox locally in the easiest
+possible way (not recommended for a production environment), then this
+is your tutorial.
+
+## Install Multipass
+
+You need to install Multipass. With Multipass you will create a Ubuntu
+VMs with NetBox installed.
+
+
+Multipass can be easily installed in Linux, Windows and macOS. See
+https://multipass.run/install for more instructions.
+
+## Launch NetBox with a Multipass instance.
+
+
+Launch a VM named `netbox` with Multipass, using the cloud init configuration that will installed NetBox. This
```

Comment:
Could the cloud init configuration be a link to the file?

Reply:
Done!